src/main/java/com/esri/arcgisruntime/toolkit/OverviewMap.java

```diff
@@ -191,4 +197,33 @@ public void setSymbol(Symbol symbol) {
   public SimpleObjectProperty<Symbol> symbolProperty() {
     return symbolProperty;
   }
+
+  /**
+   * A property containing the amount to scale the OverviewMap compared to the geoView. The default is 25.0.
+   *
+   * @return the scale property
+   * @since 100.13.0
```

Comment:
Very minor point, but did we ever release 100.13.  Was it even a release in Git?  If not then consider making it 100.14

Same consideration for other instances of 100.13 below

Reply:
We went back and forth at the time on this, but in hindsight I think that makes better sense. Amended to 100.14. Will document something for this going forward.